templates/include/sidebar.html

```diff
@@ -0,0 +1,11 @@
+<aside>
+  <nav>
+    <ul>
+      <li><a href="#" aria-label="Go to home page">Home page</a></li>
+      <li><a href="#" aria-label="Go to manufacturers list">Manufacturers</a></li>
+      <li><a href="#" aria-label="Go to cars list">Cars</a></li>
+      <li><a href="#" aria-label="Go to drivers list">Drivers</a></li>
```

Comment:
The `href` attributes are set to `#`, which means the links are not functional. Update these to point to the correct URLs within the application, such as `{% url 'taxi:index' %}` or other named URL patterns.